### ISSUE TITLE
Error when ignoring

### ISSUE THREAD

**Issue by TristanWiley** [TristanWiley/So-Chat-Commander#56]
@drobilc I think you made the ignoring system, I'm getting an error every pageload.

`Uncaught TypeError: node.querySelector is not a function`

**Comment by drobilc** [comment]
Nope, I think that is not mine 😄

**Comment by TristanWiley** [comment]
Oh, oops. That was @Arthelon

**Comment by Arthelon** [comment]
@TristanWiley Can't seem to replicate the error. Can you paste a full trace? Which browser are you using?

**Comment by TristanWiley** [comment]
```
Uncaught TypeError: node.querySelector is not a function
    at removeIgnoredUsers (chrome-extension://edapbjkpbpeooojkflphlcnijiolkplc/background.js:357:25)
    at chrome-extension://edapbjkpbpeooojkflphlcnijiolkplc/background.js:157:17
    at NodeList.forEach (native)
    at chrome-extension://edapbjkpbpeooojkflphlcnijiolkplc/background.js:156:33
    at Array.forEach (native)
    at MutationObserver.<anonymous> (chrome-extension://edapbjkpbpeooojkflphlcnijiolkplc/background.js:155:19)
```

**Comment by TristanWiley** [comment]
Chrom(ium)

**Comment by Arthelon** [comment]
Version?

**Comment by Arthelon** [comment]
It could be that your browser doesn't support NodeList's forEach [method](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach), I'll make a PR with a polyfill.

**Comment by TristanWiley** [comment]
Fixed 00715c2a6fa35b09081e5051aecd799b45874a19
